packages/documentation/src/stories/components/button/button.docs.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
 
-<Meta title="Design System/Button" />
+<Meta title="Components/Button" />
```

Comment:
I think this change should be in another PR, but it's a detail.

Reply:
It's not only a detail, it is actually doing nothing, because it is in a docs.mdx file.